src/containers/Storage/PDisk/PDisk.scss

```diff
@@ -1,28 +1,34 @@
 .pdisk-storage {
     position: relative;
 
-    width: 120px;
+    display: flex;
+    flex-direction: column;
+
+    min-width: 120px;
 
     &__content {
         position: relative;
 
         display: block;
+        flex: 1;
 
-        border-radius: 4px; // to match interactive area with disk shape
+        border-radius: 4px;
     }
 
     &__vdisks {
         display: flex;
-        // this breaks disks relative sizes, but disks rarely exceed one line
-        flex-wrap: wrap;
+        flex: 0 0 auto;
         gap: 2px;
 
         margin-bottom: 4px;
+
+        white-space: nowrap;
     }
 
     &__vdisks-item {
-        flex-basis: 3px;
-        flex-shrink: 0;
+        flex: 0 0 3px;
+
+        min-width: 3px;
```

Comment:
Your solution works well only in the case, where all pdisks have the same amount of vdisks, it's not always true.

Previously we assumed, that all pdisks should be of the same size, so `MaximumSlotsPerDisk` was introduced so it will be possible to calculate pdisk size for paginated table (something like `pDiskSize = MaximumSlotsPerDisk * VDISK_SIZE`) and use it for all pdisks in this table (even if the don't have any vdisks)

<img width="1476" alt="Screenshot 2024-12-17 at 13 46 51" src="https://github.com/user-attachments/assets/5ea2309f-467c-4b36-a824-d2164513669a" />

Reply:
fixed

Also added random vdisks count generation to pdisks in mocks